R/pkgdown.R

```diff
@@ -18,42 +18,38 @@ local_envvar_pkgdown <- function(pkg, scope = parent.frame()) {
   )
 }
 
-local_pkgdown_site <- function(path = NULL, meta = NULL, clone = FALSE, env = parent.frame()) {
-  check_bool(clone)
+local_pkgdown_site <- function(path = NULL, meta = list(), env = caller_env()) {
+  check_string(path, allow_null = TRUE)
+
+  dst_path <- withr::local_tempdir(.local_envir = env)
+  meta <- modify_list(meta, list(destination = dst_path))
 
   if (is.null(path)) {
     path <- withr::local_tempdir(.local_envir = env)
+    
     desc <- desc::desc("!new")
     desc$set("Package", "testpackage")
     desc$set("Title", "A test package")
     desc$write(file = path(path, "DESCRIPTION"))
-  }
-
-  if (clone) {
-    if (is.null(path)) {
-      cli::cli_abort("Can only clone when {.arg path} is set.")
-    } else {
-      src_paths <- dir_ls(path, recurse = TRUE)
-      is_dir <- is_dir(src_paths)
 
-      dst <- withr::local_tempdir("pkgdown", .local_envir = env)
-      dst_paths <- path(dst, path_rel(src_paths, path))
+    # Default to BS5 if template not specified
+    if (is.null(meta$template)) {
```

Comment:
since rlang is imported, this can become

```r
meta$template <- meta$template %||% list(bootstrap = 5)
```

in any case, the comment "Default to BS5..." feels useless.

Reply:
especially given the _other_ comments in the function are useful

Reply:
Does "Default to BS5 **only** if template not specified" make the comment more clear? I wrote this because I implemented it incorrectly the first time